### ISSUE TITLE
Tooltip without icon is not accessible

### ISSUE THREAD

**Issue by rophy** [IBM/carbon-components-react#1363]
Component: Tooltip

Version: I'm using v6.6.3, but looking at the src code, I think problem exists up to v6.33.0.

A tooltip with icon works correctly:

1. The icon has tabindex, so tabbing can focus on the icon.
2. When focus on the icon, tooltip shows up.

In the case of a tooltip without icon, the text div should be the focus but the div lacks tabindex, so:

1. The text div lacks tabindex, so tabbing cannot focus on the div.
2. While the div has onBlur event, it cannot receive focus, so tooltip does not show up.

Sandbox: just use the official storybook

1. Tooltip with icon: [link](http://react.carbondesignsystem.com/?knob-The%20title%20%28title%29=Section%201%20title&knob-Tooltip%20direction%20%28direction%29=bottom&knob-Trigger%20text%20%28triggerText%29=Tooltip%20label&selectedKind=Tooltip&selectedStory=default%20%28bottom%29&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel)

    - Press tab, see that the icon receives focus and shows tooltip.

2. Tooltip without icon: [link](http://react.carbondesignsystem.com/?knob-The%20title%20%28title%29=Section%201%20title&knob-Tooltip%20direction%20%28direction%29=bottom&knob-Trigger%20text%20%28triggerText%29=Tooltip%20label&selectedKind=Tooltip&selectedStory=no%20icon&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel)

    - Press tab, see that the text does not receive focus, so no tooltip.
